tests/testthat/test-bm-tpc-h.R

```diff
@@ -83,13 +88,25 @@ ORDER BY
     l_linestatus;\n"
   )
 
-  # create a new connection to ensure we're using the one that is bieng based
+  # create a new connection to ensure we're using the one that is being based
   con_one <- DBI::dbConnect(duckdb::duckdb())
-  DBI::dbExecute(con_one, paste0("CALL dbgen(sf=0.001);"))
+
+  # use it to populate the tables needed for query 1
+  get_input_func(
+    engine = "duckdb",
+    scale_factor = 0.001,
+    format = "parquet",
+    query_id = 1,
+    con = con_one
+  )
 
   query_01_func <- get_sql_query_func(1)
-  query_01_func(con = con_one)
+  expect_s3_class(query_01_func(con = con_one), "data.frame")
```

Comment:
Nice addition — this is a bit of scope creep, but I wonder if it would be possible to also test it against our answers (though we would then run the risk that duckdb breaks and doesn't actually work for this one query / at this sacle factor but that's not our fault...)

Reply:
Oh, no that's totally right — I don't have answers saved for 0.001 just 0.01, 0.1, 1, and 10. Let's leave this here — we haven't had any issues with this (and we test these when we create them in the answer creation script too, so we should see it then if we run into it...)